src/AdvancedMode.ts

```diff
@@ -184,4 +190,15 @@ function cleanCOGCPEX(buffer, tag)
 			button.textContent = "vote";
 		}
 	});
+}
+
+function cleanSHPF(buffer)
+{
+	// Remove the sort options
+	const bufferInventorySortOptions = buffer.querySelectorAll(Selector.InventorySortOptions);
+	bufferInventorySortOptions?.forEach((value) => value.remove());
```

Comment:
not sure if `remove()` is the way to go here. I don't have a lot of experience with DOM manipulation. Seems to work though.